src/ontology/config/prefixes.csv

```diff
@@ -243,14 +242,12 @@ HGNC,http://identifiers.org/hgnc/
 MEDDRA,http://identifiers.org/meddra/
 MEDGEN,http://identifiers.org/medgen/
 SCTID,http://identifiers.org/snomedct/
-UMLS,http://linkedlifedata.com/resource/umls/id/
```

Comment:
Not a duplicate. (different IRI, see my comment in issue)

Reply:
Actually it is, and it isn't--but importantly--it is! See below (3 entries):

https://github.com/monarch-initiative/mondo-ingest/blob/adc201572f76b24fa80ff907ee726932249d9b87/src/ontology/config/prefixes.csv#L240-L246